discord-lib/src/main/kotlin/org/regenagcoop/discord/Discord.kt

```diff
@@ -11,12 +13,18 @@ import org.regenagcoop.discord.model.GuildId
 /**
  * A discord client for a given guild/server.
  */
-open class Discord(
+open class Discord @OptIn(KordUnsafe::class) constructor(
     httpClient: HttpClient,
     val guildId: GuildId,
     token: String,
     val dryRun: Boolean,
-    val restClient: RestClient = RestClient(KtorRequestHandler(httpClient, token = token)),
+    val restClient: RestClient = RestClient(
+        KtorRequestHandler(
+            httpClient,
+            requestRateLimiter = ParallelRequestRateLimiter(),
```

Comment:
Kord's default [ExclusionRequestRateLimiter](https://dokka.kord.dev/rest/dev.kord.rest.ratelimit/-exclusion-request-rate-limiter/index.html) forces all requests to run serially, which is why performance wasn't better in this branch then on main. Now, even though we hit rate limits and have to do some pausing, the whole thing loads in about 15 seconds, about 4x faster than on main.